src/components/todo/Todo.scss

```
@@ -0,0 +1,30 @@
+.conteiner {
+  box-sizing: border-box;
+  display: grid;
+  width: 300px;
+  border: 1px solid black;
+  justify-content: center;
+  align-items: center;
+  font-size: larger;
+  margin-bottom: 20px;
+  padding: 10px;
+  box-shadow: 10px 5px 5px black;
+  border-radius: 50%;
+  background-color: whitesmoke;
+
+  &__done {
+    color: green;
+
+  }
+
+  &__no {
```

слово `no` мало о чем говорит
```suggestion
  &__completed {
    color: green;

  }

  &__not-completed{
```